functional/basic-attestation-on-containers/test.sh

```diff
@@ -23,6 +23,10 @@ rlJournalStart
         #create network for containers
         rlRun "limeconCreateNetwork ${CONT_NETWORK_NAME} 172.18.0.0/16"
 
+	#preparation for ssh access
```

Comment:
Given you are doing it in a test, I am thinking if it would make sense to `rlFileBackup --clean` the original dir and restore it after the test. WDYT?

Reply:
DONE.